pages/loginPage.ts

```diff
@@ -23,19 +23,23 @@ export class LoginPage {
     return await this.page.locator(`text=${text}`);
   }
 
-  async fillLoginAndPassword(): Promise<void> {
+  async login(): Promise<void> {
```

Comment:
better use performLogin, and set username and password as parameter to func, to make it reusable.